mobile/src/main/java/rs/readahead/washington/mobile/views/collect/widgets/GeoPointWidget.java

```diff
@@ -65,7 +65,7 @@ public GeoPointWidget(Context context, @NonNull FormEntryPrompt formEntryPrompt)
 
         try {
             readonlyOverride = formEntryPrompt.getIndex().getReference().getNameLast()
-                    .toLowerCase().endsWith(context.getString(R.string.tella_location_field_suffix));
+                    .toLowerCase().endsWith(context.getString(R.string.tella_location_field_prefix));
```

Comment:
Should be `startsWith`?

Reply:
I will throw this out completely, it was an earlier implementation of this feature.